### PR TITLE
Add agenda for GraphiQL WG #2

### PR DESCRIPTION
**Pull request by acao**

### CODE REVIEW

File: wg-agendas/06-18-2019.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Agenda
+
+1. Introductions
+1. Updates
+  - Access
+  - Licensing
+  - Monorepo
+1. Plugin System
+1. Language Server (LSP?) & Codemirror or Monoco?
```
Comment (BrunoScheufler) [inline wg-agendas/06-18-2019.md]: I think it was [Monaco](https://microsoft.github.io/monaco-editor/) 😄

Comment (acao) [reply wg-agendas/06-18-2019.md]: nice find, thank you!

**Comment by acao** [comment]
@BrunoScheufler yes, it was an ad-hoc decision Lee and I made last month, but yes this is where we scheduled the last GraphiQL WG meeting as well.

**Comment by acao** [comment]
To note, @schickling (of Prisma / graphql-playground) plans on being at the next GraphiQL WG meeting, and is likely very much on the same page with us in terms of the discussion today!

**Pull request merged**